playbooks/common/openshift-cluster/upgrades/v3_10/upgrade_control_plane.yml

```diff
@@ -25,6 +25,14 @@
       openshift_upgrade_min: '3.9'
       openshift_release: '3.10'
 
+- name: Run kube proxy, needed for ACI
```

Comment:
Move this below `- name: Configure components that must be available prior to upgrade` task